README.md

```diff
@@ -68,7 +70,9 @@ pip install git+https://gitlab.com/meltano/tap-gitlab.git
       "start_date": "2018-01-01T00:00:00Z",
       "ultimate_license": true,
       "fetch_merge_request_commits": false,
-      "fetch_pipelines_extended": false
+      "fetch_pipelines_extended": false,
+      "fetch_group_variables": false,
+      "fetch_project_variables": false
```

Comment:
Can you add a trailing comma here? It makes git diffs cleaner in the future.

```suggestion
      "fetch_project_variables": false,
```

Reply:
I don't think a trailing comma should apply here! This is in the README, which is specifying what the configuration map in `config.json` file should look like; trailing commas are not allowed in JSON.

However, your point about addressing trailing commas in the python code still holds; I'll fix that.